convert2rhel/subscription.py

```diff
@@ -219,7 +222,20 @@ def remove_original_subscription_manager():
     pkghandler.print_pkg_info(submgr_pkgs)
     utils.ask_to_continue()
     submgr_pkg_names = [pkg.name for pkg in submgr_pkgs]
-    utils.remove_pkgs(submgr_pkg_names, critical=False)
+
+    if system_info.id == "centos" and system_info.version.major == 8 and system_info.version.minor == 5:
+        if _SUBMGR_PKG_REMOVED_IN_CL_85 in submgr_pkg_names:
+            # The package listed in _SUBMGR_PKG_REMOVED_IN_CL_85 has been removed
+            # from CentOS Linux 8.5 and causes conversion to fail if installed on
+            # 8.4 and older for not being able to back up the package.
```

Comment:
```suggestion
            # The package listed in _SUBMGR_PKG_REMOVED_IN_CL_85 has been removed from CentOS Linux 8.5 and causes
            # conversion to fail if it's installed on that system because it's not possible to back it up.
```
I realized that aftyer we start supporting conversions of 8.4 the message would then be confusing.